nautobot/docs/user-guide/core-data-model/wireless/index.md

```diff
@@ -0,0 +1,100 @@
+# Wireless
+
+## Entity Relationship Diagram
+
+This schema illustrates the connections between related models.
+
+```mermaid
```

Comment:
Please add a screenshot of this diagram to the PR summary so that it can be reviewed